eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -113,10 +113,22 @@ public function publishUrlAliasForLocation(
         $action = 'eznode:' . $locationId;
         $cleanup = false;
 
+        // This is intentionally hardcoded and not exposed as:
+        // 1. we don't implement this configuration option
+        // 2. such option should have no place here
+        // This solution is temporary until we can break data compatibility with Legacy Stack.
+        $contentTreeRootLocationId = 2;
```

Comment:
While I agree with the points above, I'd still use at least use a property. I'd even maybe go as far as expose this property (`array $options` ?), BUT warn, maybe even with an exception, that changing this might trigger the end of the universe.